tests/unit/test_requirements_generation.py

```
@@ -0,0 +1,185 @@
+"""Tests for requirements.txt generation from pyproject.toml."""
```

Ruff (0.12.2)

File `tests/unit/test_requirements_generation.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)